### PR TITLE
Updates tsapi to handle the new way TS handles file paths. Fix #223

### LINKED ISSUE #223: Sometimes gulp-typescript doesn't generate any files when using typescript@next
Now Typescript at 1.8.0-dev.20151028, with gulp-typescript, I've been able to repro a bug where gulp-typescript just doesn't output anything.

### PR DESCRIPTION
**Pull request by kenotron**
Here's a PR that will fix #223.

### CODE REVIEW

File: lib/tsapi.ts [hunk 1]
```diff
@@ -147,3 +151,13 @@ export function transpile(typescript: TypeScript14 | TypeScript15, input: string
...
 }
+
+
+export function getNormalizedAbsolutePath(typescript: TypeScript18 | TypeScript14 | TypeScript15, fileName: string, currentDirectory: string) {
+	// Prior to Typescript 1.8.0-dev.20151028, this method did not exist, so whatever is passed in from fileName is passed through.
+	if (!(<TypeScript18> typescript).getNormalizedAbsolutePath) {
+		return fileName;
+	}
+
+    return (<TypeScript18> typescript).getNormalizedAbsolutePath(fileName, currentDirectory);
```
Comment (ivogabe) [inline lib/tsapi.ts]: Can you change spaces to tabs here?

Comment (kenotron) [inline lib/tsapi.ts]: Okay this is done. 2nd Commit to have that change :)

**Comment by robertbaker** [comment]
@ivogabe Although I can argue one should use spaces (the accepted norm now) over tabs, I won't.

But we need this merged and tagged for release!

**Comment by kenotron** [comment]
Already fixed in the latest typescript@next. Retracting this PR.

**Pull request closed without merging**